exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -320,6 +321,19 @@ func timestampToFloatSeconds(ts pcommon.Timestamp) float64 {
 	return float64(ts) / float64(time.Second)
 }
 
+func addSpecialAttributes(attributes map[string]pcommon.Value, indexedAttrs []string, unfilteredAttributes pcommon.Map) map[string]pcommon.Value {
+	for _, name := range indexedAttrs {
+		// Index attributes that have been filtered out before if explicitly added
+		_, filteredIn := attributes[name]
```

Comment:
Nit:
I found variable names like filteredIn a little confusing on their purpose. Could we use something like currentValue, overrideValue, hasOverrideValue, ect.

Reply:
addressed